app/routes/users/components/StudentConfirmation.tsx

```diff
@@ -1,206 +1,229 @@
+                <p>{authRes.detail}</p>
+              </>
+            )}
+          </Card>
+        )}
+
+        <Button success onClick={() => performStudentAuth()}>
+          Verifiser med FEIDE
+        </Button>
+        {isStudent != null && (
+          <p className={styles.infoText}>
+            Du har allerede verifisert din status. Dersom du har byttet studie
+            og ønsker å bli medlem av Abakus, kan du verifisere deg på nytt og
+            vi vil oppdatere statusen din dersom du er registrert riktig på
+            StudentWeb.
+          </p>
+        )}
+
+        <Modal
+          show={showMemberModal}
+          contentClassName={styles.membershipModalContent}
+          onHide={() => setShowMemberModal(false)}
+        >
+          <Card info>
+            <Flex column>
+              <h4>Din studentstatus ble godkjent!</h4>
+              <p>
+                Du har blitt satt i gruppen: <b>{authRes?.grade}</b>
+              </p>
             </Flex>
-            <Field
-              name="studentIsMemberYes"
-              label="Ja"
-              component={RadioButton.Field}
-              inputValue="true"
-            />
-            <Field
-              name="studentIsMemberNo"
-              label="Nei"
-              component={RadioButton.Field}
-              inputValue="false"
-            />
-          </RadioButtonGroup>
-          <Field
-            name="captchaResponse"
-            fieldStyle={{
-              width: 304,
-            }}
-            component={Captcha.Field}
-          />
-          <Button submit disabled={disabledButton}>
-            Verifiser
-          </Button>
-        </Form>
+          </Card>
+          <h2>Vil du bli medlem av Abakus?</h2>
+          <div>
+            <p className={styles.infoText}>
+              Alle som går Kommunikasjonsteknologi & Digital Sikkerhet eller
+              Datateknologi kan bli medlem av Abakus. Du må bli medlem for å
+              kunne delta på arrangementer, kurs og annet Abakus tilbyr. Vi
+              anbefaler alle nye studenter å melde seg inn. Du kan lese mer om{' '}
+              <a
+                href="https://abakus.no/pages/info-om-abakus"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Abakus
+              </a>{' '}
+              og{' '}
+              <a
+                href="https://statutter.abakus.no#medlemskap"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                medlemskapet.{' '}
+              </a>
+              Det koster ingenting å være medlem av Abakus.
+            </p>
+            <p className={styles.infoText}>
+              Du kan alltids endre medlemskapet ditt på dine{' '}
+              <Link to="/users/me/settings/profile">innstillinger</Link>.
+            </p>
+          </div>
+
+          <Flex row>
```

Comment:
```suggestion
          <Flex>
```
`row` is default and not an accepted prop